main/src/main/res/values/strings.xml

```diff
@@ -7,79 +7,79 @@
     <string name="address">Server Address:</string>
     <string name="port">Server Port:</string>
     <string name="location">Location</string>
-    <string name="cant_read_folder">Unable to read directory</string>
+    <string name="cant_read_folder">Could not read directory</string>
     <string name="select">Select</string>
     <string name="cancel">Cancel</string>
     <string name="no_data">No Data</string>
     <string name="useLZO">LZO Compression</string>
     <string name="client_no_certificate">No Certificate</string>
     <string name="client_certificate_title">Client Certificate</string>
     <string name="client_key_title">Client Certificate Key</string>
-    <string name="client_pkcs12_title">PKCS12 File</string>
+    <string name="client_pkcs12_title">PKCS 12 File</string>
```

Comment:
Also PKCS12 is like file a type, so writing it together is better.

Reply:
It used to be "PCKS 12" http://www.foo.be/docs-free/opensst/ref/pkcs-12v1.pdf
and is now `PCKS #12` https://datatracker.ietf.org/doc/html/rfc7292 by the looks of it.

As long as it isn't both that is better, but I think the latter is the more correct one.
Sorry for getting this wrong.